src/app/components/home/home.component.scss

```diff
@@ -3,7 +3,7 @@
 
 .home-container {
   overflow-x:hidden;
-  overflow-y:scroll;
+  //overflow-y:scroll;
```

Comment:
Please don't comment and commit your code changes. It should be either present or removed (if there is any specific reason for the code comment, then it should be written properly).

Reply:
okay, I will keep in mind.  This the issue due to which the main is using y-axis scroller by default.
Even when there is no overflow.